packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -134,7 +140,7 @@ const diffusers_default = (model: ModelData) => [
 
 pipe = DiffusionPipeline.from_pretrained("${model.id}")
 
-prompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k"
+prompt = "${get_prompt_from_diffusers_model(model)}"
```

Comment:
following the comment [here](https://github.com/huggingface/huggingface.js/pull/909/files#r1764685115)

it should be something like:
```ts
const defaultPrompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k";
prompt = "${get_prompt_from_diffusers_model(model) ?? defaultPrompt}"
```

saying that if example prompt does not exist in model card, it should fallback to `defaultPrompt`